tests/e2e/scale/test_osd_node_balancing.py

```diff
@@ -0,0 +1,193 @@
+    logging.info(f"{osd_running_nodes} associated machineset ")
+    node_obj = ocp.OCP(kind="node")
+    # This next section makes a new node on which OSD pods can run
+    for mnames in range(0, NUM_NODE_ADDS):
+        # This next section makes a new node on which OSD pods can run
+        new_node = add_new_node_and_label_it(
+            machineset_names[mnames], mark_for_ocs_label=False
+        )
+        node_obj.add_label(
+            resource_name=new_node[0], label=constants.OPERATOR_NODE_LABEL
+        )
+    collect_stats("Three nodes have been added")
+
+
+def increase_osd_capacity():
+    """
+    Add three osds to the cluster.  Update the report file.
+    """
+    # This next section increases the number of OSDs on each node by one.
+    osd_running_nodes = get_osd_running_nodes()
+    osd_size = storage_cluster.get_osd_size()
+    logging.info(f"Adding one new set of OSDs. osd size = {osd_size}")
+    storagedeviceset_count = storage_cluster.add_capacity(osd_size)
+    if type(storagedeviceset_count) == bool:
+        logging.info("storeagedeviceset is boolean")
+    else:
+        logging.info(f"storeagedeviceset is {storagedeviceset_count}")
+        logging.info("Adding one new set of OSDs was issued without problems")
+        new_osd_count = len(osd_running_nodes) + 3
+        pod_obj = ocp.OCP(kind=constants.POD, namespace="openshift-storage")
+        for new_osd_list in TimeoutSampler(
+            120, 30, lambda: pod_obj.get(selector=constants.OSD_APP_LABEL)["items"]
+        ):
+            if len(new_osd_list) == new_osd_count:
+                break
+    ceph_health_check(tries=30, delay=60)
+    collect_stats("OSD capacity has been increased")
+
+
+def collect_stats(action_text):
+    """
+    Write the current configuration information into the REPORT file.
+    This information includes the osd, nodes and which osds are on which
+    nodes.  The minimum and maximum numbers of osds per node are also
+    computed and saved.  If this is the final call to collect_stats
+    (action_text parameter is FINAL_REPORT), then the data collected
+    in the REPORT file is also displayed in the log.
+
+    Args:
+        action_text -- Title of last action taken
+                (usually adding nodes or adding osds)
+    """
+    pod_obj = ocp.OCP(
+        kind=constants.POD, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
+    )
+    osd_list = pod_obj.get(selector=constants.OSD_APP_LABEL)["items"]
+    node_stats = {}
+    for osd_ent in osd_list:
+        try:
+            osd_node = osd_ent["spec"]["nodeName"]
+        except KeyError:
+            continue
+        if osd_node in node_stats:
+            node_stats[osd_node].append(osd_ent)
+        else:
+            node_stats[osd_node] = [osd_ent]
+    osds_per_node = []
+    for entry in node_stats:
+        osds_per_node.append(len(node_stats[entry]))
+    wnodes = get_nodes(constants.WORKER_MACHINE)
+    for wnode in wnodes:
+        logging.info(wnode.name)
```

Comment:
`wnode.name` doesn't provide any valuable info. What is the intention of adding this to the logs?

Reply:
I'll remove the log statement.